README.md

```diff
@@ -155,32 +154,41 @@ https://github.com/mrts/docker-qt-cmake-gtest-valgrind-ubuntu/blob/master/Docker
 
 ### Windows
 
-- Download Visual Studio 2019 community installer from https://visualstudio.microsoft.com/ and install *Desktop C++ Development*
+- Download Visual Studio 2019 community installer from https://visualstudio.microsoft.com/ and install _Desktop C++ Development_
 - Download WIX toolset from https://wixtoolset.org/ and install version 3.11.2
 - Download and install Git for Windows from https://git-scm.com/download/win
-- Install *vcpkg* by running the following commands in Powershell:
+- Install _vcpkg_ by running the following commands in Powershell:
 
       git clone https://github.com/microsoft/vcpkg.git C:\vcpkg
       cd C:\vcpkg
       .\bootstrap-vcpkg.bat
       .\vcpkg integrate install
 
-- Install *Google Test*, *OpenSSL* and *Qt* with *vcpkg*:
+- Install _Google Test_ and _OpenSSL_ with _vcpkg_:
+
+      .\vcpkg install --recurse --triplet x64-windows --clean-after-build gtest openssl
 
-      .\vcpkg install gtest:x64-windows openssl:x64-windows qt5-base[core]:x64-windows
+- Install _Qt_:
+
+      set Qt5_DIR=C:\qt
+      python -m venv venv
```

Comment:
Where comes python? another requirement?

Reply:
This is just a recommendation in README, not mandatory. But you are right, it would make sense to explain this in more detail and add instructions for installing Python.

Reply:
I added instructions for manual Qt download and de-emphasized aqt, also linked to Python installer. Does this look good now?